IDEAS/Buildings/Components/Shading/Screen.mo

```diff
@@ -1,13 +1,26 @@
 within IDEAS.Buildings.Components.Shading;
 model Screen "Controllable exterior screen"
-  extends IDEAS.Buildings.Components.Shading.Interfaces.PartialShading(
-                                                             final controlled=true);
+  extends IDEAS.Buildings.Components.Shading.Interfaces.PartialShadingDevice(
+    TSha = TShaScreen,
+    TDryBul_internal = limiter.y*TSha + (1-limiter.y)*Te_internal,
+    epsSw_shading = 1 - shaCorr,
+    final controlled = true,
+    TEnvExpr(y = TEnv_screen),
+    TeExpr(y = TDryBul_internal));
 
-  parameter Real shaCorr=0.24 "Shortwave transmittance of shortwave radiation";
+  parameter Real shaCorr(min=0, max=1) = 0.24
+    "Shortwave transmittance of the screen";
 
 protected
-  Modelica.Blocks.Nonlinear.Limiter limiter(uMax=1, uMin=0);
-
+  Modelica.Units.SI.Temperature TEnv_screen = limiter.y*TSha + (1-limiter.y)*TEnv_internal;
```

Comment:
A documentation string should be provided.

Reply:
yup